### DISCUSSION TITLE
Node out of Frame

### DISCUSSION THREAD

**Discussion by Kagiro-K** [talmolab/sleap#1245] [Help!] [answered]
Hi! 
I'm relatively new to using SLEAP and still learning a lot!
Is there a way to move a node that is out of the frame entirely? Sometimes nodes will get predicted "off screen" and I can't figure out how to move them back into view.

Thank you!

![image](https://user-images.githubusercontent.com/126974537/228324062-7c1146d7-8bb0-450f-82b1-2e8e4661c6e3.png)

**Comment by roomrys** [comment, marked as answer]
Hi @Kagiro-K,

Hmm, I thought we had implemented something to ensure this does not happen... but I will need to double check whether this fix is in 1.2.9+. If it is not, then this problem will need to be readdressed. 

For now, you can move the entire instance over by holding Alt + dragging any node.

Thanks,
Liezl

**Comment by Kagiro-K** [comment]
Wow thank you so much!!

I thought I tried every combination of holding control, shift, or alt, but I guess I must have not tried alt by itself!!

________________________________

**Reply by roomrys** [reply]
Ha pretty good guesses!

**Comment by Kagiro-K** [comment]
I really appreciate the super quick response thank you so much!
It's incredible what your lab has created.
Thank you for sharing it with the community!

________________________________
